src/command/CommandManager.ts

```diff
@@ -1,5 +1,5 @@
-import Player from "../player";
-import Prismarine from "../prismarine";
+import Player from "../player/Player";
```

Comment:
I’m not sure how I feel about /player/Player, doesn’t index make more sense here?

Reply:
I like the idea of `/player/Player` because it follows the class name to class file orientation, index may confuse intellisense depending on your IDE